src/errors.rs

```diff
@@ -85,20 +83,6 @@ impl de::Error for Error {
     }
 }
 
-#[allow(use_debug)]
-impl From<Error> for neon::result::Throw {
-    fn from(err: Error) -> Self {
-        if let ErrorKind::Js(_) = *err.kind() {
-            return neon::result::Throw;
-        };
-        let msg = format!("{:?}", err);
-        unsafe {
-            neon_runtime::error::throw_error_from_utf8(msg.as_ptr(), msg.len() as i32);
-            neon::result::Throw
-        }
-    }
-}
-
```

Comment:
@joshuef Thanks for doing the work of upgrading. I just started using this. Not having this trait means we can't use the `?` operator for handling Result types. Is there a reason we cannot have this trait?

Reply:
It's explained here: https://docs.rs/neon/0.7.1/neon/result/struct.Throw.html

Kinda unfortunate though, breaks the `export!` macro. It uses `?` and should be changed to handle errors properly. Couldn't tell you exactly how though.